codebuild_specs/build_windows.yml

```diff
@@ -4,10 +4,6 @@ env:
 phases:
   build:
     commands:
-      - cd scripts
-      - yarn
```

Comment:
might need to add these back in (lines 7, 8); appears that the scripts cache is missing in subsequent steps